lib/annotations/access.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+const { isProse } = require('./../utils/prose');
+
+module.exports = (env) => {
+  const defaultPrivatePrefixTest = RegExp.prototype.test.bind(/^[_-]/);
+  const baseAccessFn =
+    // eslint-disable-next-line global-require
+    require('sassdoc/dist/annotation/annotations/access').default;
+  const baseAccess = baseAccessFn();
+
+  return {
+    ...baseAccess,
+
+    autofill(item) {
+      if (isProse(item)) {
+        return undefined;
+      }
+
+      if (item.access !== 'auto') {
+        return undefined;
+      }
+
+      if (env.privatePrefix === false) {
+        return undefined;
+      }
+
+      let testFunc = defaultPrivatePrefixTest;
+
+      if (typeof env.privatePrefix !== 'undefined') {
+        testFunc = RegExp.prototype.test.bind(new RegExp(env.privatePrefix));
+      }
```

Comment:
At the point where Herman needs to override this `autofill` fn, we don't have access to the actual `env` yet. Because this annotation relies on values from the runtime `env`, we need to pull this annotation into our repo and rewrite it (instead of just augmenting the SassDoc annotation).